Directory.Build.props

```diff
@@ -1,19 +1,20 @@
 <Project>
-	<PropertyGroup>
-		<TargetFrameworks>net5.0;net6.0;net7.0</TargetFrameworks>
-		<Platform>x64</Platform>
-		<GenerateAssemblyInfo>true</GenerateAssemblyInfo>
-		<Nullable>enable</Nullable>
-		<NullableContextOptions>enable</NullableContextOptions>
-		<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-		<LangVersion>11</LangVersion>
-		<Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-		<DebugType Condition=" '$(Configuration)' == 'Debug' ">full</DebugType>
-		<DebugType Condition=" '$(Configuration)' == 'Release' ">pdbonly</DebugType>
-		<OutputPath>$(MSBuildThisFileDirectory)\bin\$(Configuration)\$(MSBuildProjectName)\</OutputPath>
-		<RootNamespace>EventStore.Client</RootNamespace>
-		<UseLinkBase>true</UseLinkBase>
-		<GrpcPackageVersion>2.49.0</GrpcPackageVersion>
-		<GrpcToolsPackageVersion>2.50.0</GrpcToolsPackageVersion>
-	</PropertyGroup>
+    <PropertyGroup>
```

Comment:
did tabs get changed to spaces here?

Reply:
Will fix on another PR of "cosmetic" nature. And  yes I'm struggling with tabs a lot XD